content/blog/getting-started-with-hpe-greenlake-api-for-virtualization.md

```diff
@@ -0,0 +1,412 @@
+        "name": "",
+        "resourceUri": "/data-services/v1beta1/async-operations/de6d9725-xxxx-xxxx-xxxx-f40ad05f879f",
+        "type": "task"
+    },
+    "services": [
+        "private-cloud-business-edition"
+    ],
+    "sourceResource": {
+        "name": "RonD-deploy-CSP-1",
+        "resourceUri": "/api/v1/csp-machine-instances/d4c46c6e-xxxx-xxxx-xxxx-ef52d77f0b16",
+        "type": "AWS Instance"
+    },
+    "startedAt": "2024-03-27T22:37:20.509668598Z",
+    "state": "SUCCEEDED",
+    "subtreeTaskCount": 1,
+    "suggestedPollingIntervalSeconds": 30,
+    "type": "task",
+    "updatedAt": "2024-03-27T22:37:37.688700579Z",
+    "userId": "ronald.dharma@hpe.com"
+}
+```
+
+### Wow… that was so cool. What about provisioning  a VMware Datastore on-premises?
+
+Here is another example of using the HPE GreenLake API for virtualization to provision a datastore out of an HPE disaggregated Hyperconverged Infrastructure (dHCI) system for on-premise deployment. The HPE dHCI instance had already been onboarded into the HPE GreenLake for Private Cloud Business Edition service (PCBE).  The [API](https://developer.greenlake.hpe.com/docs/greenlake/services/virtualization/public/openapi/virtualization-public-v1beta1/operation/CreateDS/) that is used for the datastore provisioning is `POST {baseUrl} /virtualization/v1beta1/datastores`, which also required a definition of this JSON structure with values in the **body (Payload)**. The information about this request JSON body is presented in the figure below. 
+
+![](/img/create-datastore-on-dhci.png)
+
+*T﻿he above figure display the required body (Payload) to deploy datastore into a hyper-converged instance with VMware.*
+
+This virtualization API incorporates the virtual machine provisioning policy that is part of the HPE GreenLake for Private Cloud Business Edition. Use the HPE GreenLake for Private Cloud Business Edition API `GET /private-cloud-business/v1beta1/vm-provisioning-policies` to obtain the **provisioningPolicyId** and enter it into the Body JSON key-pair values.
+
+![](/img/get-provisioning-policy-for-pcbe-in-dhci.png "get the provisioning policy for deploying datastores in a dHCI using PCBE")
+
+Using another API from the virtualization API such as **GET {baseUrl}/virtualization/v1beta1/hypervisors-clusters** to obtain the **HyperClusterId** is shown below.
+
+![](/img/obtain-the-cluster-id-for-deployment-at-dhci-using-pcbe.png "Obtain the cluster id which is required to deploy a datastore in a dHCI using PCBE")
+
+And using the legacy API such as `GET /api/v1/storage-systems/device-type2` to obtain the **Storage System Id** is shown below.
+
+![](/img/obtain-the-storage-system-id-using-the-legacy-api.png "legacy API to display the storage system Id")
+
+Finally, all the above values were entered into the JSON body below as part of the Virtualization API to create a datastore in the designated cluster using the attached storage system.
```

Comment:
Two ways to handle this one. Either:
1) Finally, all the above values are entered into the JSON...
2) Finally, I entered all the above values into the JSON...

Based on your next paragraph, I would choose #2.